eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/FeeProvider.scala

```diff
@@ -27,7 +27,7 @@ trait FeeProvider {
   def getFeerates: Future[FeeratesPerKB]
 }
 
-case object CannotRetrieveFeerates extends RuntimeException("cannot retrieve feerates: channels may be at risk")
+case object CannotRetrieveFeerates extends RuntimeException("cannot retrieve feerates: ensure bitcoind serves rates and restart eclair")
```

Comment:
nit: I would still keep `channels may be at risk`, to make sure node operators understand this is quite a serious issue (if it happens on mainnet).
```suggestion
case object CannotRetrieveFeerates extends RuntimeException("cannot retrieve feerates, channels may be at risk: ensure bitcoind estimatesmartfee correctly returns feerates and restart eclair")
```